### ISSUE TITLE
Ripley attacks when moving head and crouch doesn't work

### ISSUE THREAD

**Issue by NotADeveloperAtAll** [Nibre/MotherVR#173]
Hi!
I'm using the Valve index with and Xbox controller and I have an issue where everytime I move my head it triggers the attack control. Also, the crouch buttn doesn't seem to work.

EDIT: Nevermind, it works if I turn on the index controllers.

**Comment by Gonzaxpain** [comment]
yeah, that's a new problem, it does not work with the index controllers off. I hope it will be fixed some day.

**Comment by jerahnova** [comment]
Same issue here. Using the index and my gamepad (ds4) only works when the index controllers are connected.

**Comment by wildw1ng** [comment]
same here

**Comment by SimplexPL** [comment]
Can confirm this issue, it was introduced in 0.8.0, not present in 0.6.0.

**Comment by linkocrn** [comment]
Have same issue

**Comment by SimplexPL** [comment]
You need to keep you vr controllers turned on.

**Comment by linkocrn** [comment]
Yeah it helps thanks. Though from time to time I seem to lose control of a specific button completely. Then it came back after a cutscene
